CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
```

Comment:
I've added this to unreleased for now, but we might want to just bump the version to unblock us. Wdyt @kevindew?

Reply:
For preference we'd follow the normal release process: https://github.com/alphagov/govuk_publishing_components/blob/main/docs/publishing-to-rubygems.md

Reply:
Yup agree on normal release process, can be part of next batch release